dotnet/Capstone/Capstone.csproj

```diff
@@ -6,6 +6,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <PackageReference Include="MaxMind.GeoIP2" Version="5.1.0" />
```

Comment:
Do you know where this came from? If so, what does it do?

Reply:
I have no idea.